tests/unit/conftest.py

```diff
@@ -172,9 +177,16 @@ def users(app, db):
                                       password='tester', active=True)
         user2 = datastore.create_user(email='test@zenodo.org',
                                       password='tester2', active=True)
+        user_admin = datastore.create_user(email='admin@zenodo.org',
```

Comment:
Related to the password hashing question below. I think here the users need to be created using [``invenio_accounts.testutils.create_test_user()``](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/testutils.py#L50) - then the password will be set correctly.